mock/py/mockbuild/backend.py

```diff
@@ -778,7 +826,7 @@ def install_build_results(self, results):
                 results_bindmount.mount()
             pkgs = [pkg for pkg in results if not pkg.endswith("src.rpm")]
             try:
-                self.install(*pkgs)
+                self.buildroot.install(*pkgs)
```

Comment:
Is this change needed?

Reply:
Not necessary, but I find it more intuitive. Rather Buildroot.install() than Command.install().

Reply:
But it's not entirely equivalent.  The `Command.install` prints additional log output.  If we really want to change the output I'd prefer to have this in separate commit that is saying that we don't want those log entries.

Reply:
Ok, this eventually broke `make --install`, as I was right initially - the Commands.install is used in mock.py.